coda_v2_python_client/firebase_client_wrapper.py

```diff
@@ -0,0 +1,36 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials
+from firebase_admin import firestore
+
+
+class FirebaseClientWrapper:
+    def __init__(self, client):
+        """Inits FirebaseClientWrapper
+
+        Args:
```

Comment:
Python has multiple valid schemes for filling out docstrings, but I'd suggest using the same format as we use everywhere else at AVF for consistency

Reply:
thanks, noted.